### DISCUSSION TITLE
esp32 s3 devkit c n8r8 not displaying on ili9488 or ili9341

### DISCUSSION THREAD

**Discussion by ripjohnbrown1859** [Bodmer/TFT_eSPI#2871] [Q&A - General]
im struggling to get my new s3 devkit c to work with any of my displays, i got it specifically for the 8mb of psram to use with this library. 
I originally just copied the code that worked on the esp32 wroom module and pasted it into a new platform io s3 devkit project with the needed libraries and got a "store prohibited" crash message, which im not sure is related to this lib, im now just trying to get the ili 9488 spi display to print "hello world" and i cant. i cant even get it to display anything but a white screen. i initially figured it was a wiring issue, but i triple checked the wiring and its all correct. here are my pin assignments.
```
#define TFT_CS   10 //     10 or 34 (FSPI CS0) 
#define TFT_MOSI 11 //     11 or 35 (FSPI D)
#define TFT_SCLK 12 //     12 or 36 (FSPI CLK)
#define TFT_MISO 13 //     13 or 37 (FSPI Q)
// Use pins in range 0-31
#define TFT_DC    7
#define TFT_RST   6
#define TOUCH_CS 16 // Optional for touch screen
```
heres the code
```
#include <Arduino.h>
#include <TFT_eSPI.h>
#include <SPI.h>
TFT_eSPI tft = TFT_eSPI();
 void setup()
 {
   Serial.begin(115200);
   Serial.println("Hello World!");
    tft.init();
    Serial.println("tft init");
    tft.setRotation(1);
    tft.fillScreen(TFT_WHITE);
    tft.setTextColor(TFT_BLACK);
    tft.setTextSize(1);
    tft.setCursor(0, 0);
    tft.println("Hello World!");

 }
 void loop(){

 }
```

**Comment by Bodmer** [comment]
Run the Read_User_Setup sketch, look at the serial output and check all settings are as you expect.
